src/components/common/Avatar.jsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo } from 'react';
+import { Avatar as MUIAvatar, Typography } from '@mui/material';
+
+export const Avatar = (props) => {
+	const { title, size = 25, image } = props;
+
+	console.log('props', props);
```

Comment:
we should be removing or commenting out console.logs before we release them to any environment.